index.html

```diff
@@ -103,20 +132,17 @@
 
 tasks
   .create({label: "First item", done: false})
-  .then(_ => tasks.sync())
-  .then(_ => tasks.list())
-  .then(records => { console.log(records)});</code>
-        </pre>
+  .then(_ => tasks.sync());</code></pre>
```

Comment:
I like it more focused on the feature highlighted, but I'm not against showcasing list() really. Your call.